pkg/config/setup/process.go

```diff
@@ -227,9 +224,11 @@ func loadProcessTransforms(config pkgconfigmodel.Config) {
 		} else if enabled, _ := strconv.ParseBool(procConfigEnabled); enabled { // "true"
 			config.Set("process_config.process_collection.enabled", true, pkgconfigmodel.SourceAgentRuntime)
 			config.Set("process_config.container_collection.enabled", false, pkgconfigmodel.SourceAgentRuntime)
+			config.Set("process_config.enabled", "true", pkgconfigmodel.SourceAgentRuntime)
 		} else { // "false"
 			config.Set("process_config.process_collection.enabled", false, pkgconfigmodel.SourceAgentRuntime)
 			config.Set("process_config.container_collection.enabled", true, pkgconfigmodel.SourceAgentRuntime)
+			config.Set("process_config.enabled", "disabled", pkgconfigmodel.SourceAgentRuntime)
```

Comment:
Q: Do we use `disbale` string is a standard for the agent to express a feature is disable? Do we support other values?

Reply:
This is purely keeping the current behavior. We don't have any guidance for this king of behavior and each setting was handled independently by each team.
I kept the behavior the same to be backward compatible.